debezium-core/src/main/java/io/debezium/config/InvalidConfigurationException.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.config;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import org.apache.kafka.common.config.ConfigValue;
+
+/**
+ * Details about an invalid {@link Configuration}.
+ * 
+ * @see Configuration#validateAndThrow(Field.Set)
+ * @see Configuration#validateAndThrow(io.debezium.config.Field.Set, String)
+ * @author Randall Hauch
+ */
+public class InvalidConfigurationException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    private final Map<String, ConfigValue> invalidConfigValues;
```

Comment:
`org.apache.kafka.common.config.ConfigValue` is not `Serializable`, so if the intent of this exception is to be `Serializable`, I can see a problem with holding on to this `Map`

Perhaps converting the `Map` values to something else or simply making it `transient`, depending on what the intent it is a better alternative